notion-core/src/test/scala/com/ambiata/notion/core/LocationSpec.scala

```diff
@@ -13,5 +16,9 @@ class LocationSpec extends Specification { def is = s2"""
    ${ fromUri("file:/hello/world") ==== LocalLocation("/hello/world").right }
    ${ fromUri("s3://hello/world") ==== S3Location("hello", "world").right }
 
+ A Location can be serialized/deserialized to/from Json $json
+
 """
+
+  def json = prop(implicitly[CodecJson[Location]].codecLaw.encodedecode _)
```

Comment:
@charleso Was this encoded somewhere in ivory or somewhere else - can't remember?